### DISCUSSION TITLE
[Feature request] For Photoshop/PSD files it would be nice to identify layer groups

### DISCUSSION THREAD

**Discussion by pdfkungfoo** [ImageMagick/ImageMagick#6699] [Contribute]
For Photoshop (PSD) files ImageMagick can identify layer names (`'%[label]'`), layer blend modes (`'%[compose]'`), compression type (`'%[compression]'`), but not layer groupings.

It would be nice if it could also identify layer groupings.

**Comment by urban-warrior** [comment]
Tell us how we can identify layer grouping and we will try to export it as you suggest.  Can you point us to some documentation on where groups are defined in the PSD image format?

**Comment by bioworldmerch** [comment]
Layer groups are just top level layers with layers inside of them.
